packages/cicero-core/src/template.js

```diff
@@ -156,9 +156,6 @@ class Template {
         if(this.parserManager.getTemplate()) {
             content.grammar = this.parserManager.getTemplate();
         }
-        if(this.authorSignature !== null){
```

Comment:
@jeromesimeon exept for template studio/library, what would be the use cases to include the signature file in the content hash?